src/hgvs/extras/babelfish.py

```diff
@@ -17,12 +25,17 @@ def _as_interbase(posedit):
 
 
 class Babelfish:
+    """ The plan here is to get this into biocommons HGVS via pull request
```

Comment:
Once merged and released, this comment no longer applies, right? If so, plea remove.

Reply:
ha ha, whoops yeah I copy/pasted it into my code to test it for a while